report_alternative_layout/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,8 @@
+Go to *Settings > Technical > Reporting > Reports*, and open a report of your choice, and
+update the following fields as necessary.
+
+- Apply Alternative Layout: If selected, the alternative layout will be applied in the
+  printed report.
+- Show Commercial Partner: If selected, the commercial partner of the document partner
+  will show in the report output (instead of the document partner).
+- Set the Paper Format field to 'A4 - no company address in header.'
```

Comment:
To be consistent with style, and also to make it in the right order (important ones first).

```suggestion
- Paper Format: Select 'A4 - no company address in header' if Apply Alternative Layout is selected.
- Show Commercial Partner: If selected, the commercial partner of the document partner
  will show in the report output (instead of the document partner).
```